R/ci.R

```diff
@@ -107,7 +129,7 @@ use_coverage <- function(type = c("codecov", "coveralls")) {
 }
 
 use_codecov_badge <- function() {
-  check_uses_github()
+  uses_github()
```

Comment:
I think `check_uses_github()` does the right thing here?

Reply:
hm, yeah. Idk why I made this change.